src/components/tab/SearchInput.vue

```diff
@@ -130,6 +130,19 @@ export default {
 			}
 			return ''
 		},
+		getNoOptionText() {
+			switch (this.noOptionTextState) {
+			case NO_OPTION_TEXT_STATE.START_TYPING :
+				return t('integration_openproject', 'Start typing to search')
```

Comment:
```suggestion
				return t('integration_openproject', 'Start typing to search')
```

![Screenshot from 2024-03-14 10-54-49](https://github.com/nextcloud/integration_openproject/assets/61624650/e5112e62-37d9-43c4-b2c0-eda1e1dd3ac3)

Two messages that seem to be saying the same thing